activity/actreply/activity.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "flogo-response",
+  "type": "flogo:activity",
+  "ref": "github.com/TIBCOSoftware/flogo-contrib/activity/actreply",
```

Comment:
Same comment as before, we should be consistent between the name flogo-response and activity folder actreply